packages/ds-medicare-gov/docs/src/pages/components/HHSLogo/HHSlogo.example.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { HHSLogo } from '@design-system';
+import { HHSLogo } from '@cmsgov/design-system';
```

Comment:
This is intentionally `@design-system`. It's aliased in webpack

Reply:
gotcha, my vscode intellisense marks it as unknown but it works fine on compilation